src/components/table/Tr/Tr.tsx

```diff
@@ -15,5 +15,14 @@ export const Tr = forwardRef<HTMLTableRowElement, TrProps>(({ children, ...props
 })
 
 const StyledTr = styled.tr`
-  box-shadow: 0 -1px 0 ${({ theme }) => theme.colors.border.primary};
+  // display: block needed for box-shadow in safari, can't be set on table row directly
+  &:before {
```

Comment:
Two colons seems to be the preferred way: https://developer.mozilla.org/en-US/docs/Web/CSS/::before

```suggestion
  &::before {
```